example/discord_bot/basic_prefix.py

```diff
@@ -0,0 +1,34 @@
+import discord
+
+import vacefron
+
+from discord.ext import commands
+
+intents= discord.Intents.default()
+
+
+class Bot(commands.Bot):
+    def __init__(self, **kwargs):
+        super().__init__(command_prefix="!", intents=intents, **kwargs)
+        self.vac_api= vacefron.Client()
+        
+
+    #Optional
+    async def close(self):
+        await self.vac_api.close()
+        await super().close()
+
+bot= Bot()   
```

Comment:
Same as above, space between name and `=`
```suggestion
bot = Bot()
```

Reply:
This change hasn't been committed, why was this resolved?